### ISSUE TITLE
PyYAML hook missing?

### ISSUE THREAD

**Issue by mar10** [marcelotduarte/cx_Freeze#1541]
Hi, 

I try to build an MSI installer and have problems with the PyYAML package (package name [PyYAML](https://pypi.org/project/PyYAML/), but imported as `import yaml`).

There seems to be a fix for `cx_Freeze/hooks.py` from 2021-06-23:  
    https://github.com/marcelotduarte/cx_Freeze/pull/1124/files
with this change:
```py
def load_yaml(finder: ModuleFinder, module: Module) -> None:
    """PyYAML requires its metadata"""
    module.update_distribution("PyYAML")
```

However this file does not exist in the current `cx_Freeze/` directory:  
    https://github.com/marcelotduarte/cx_Freeze/tree/main/cx_Freeze
instead, hooks seem to be implemented in a sub folder here:
    https://github.com/marcelotduarte/cx_Freeze/blob/main/cx_Freeze/hooks/__init__.py
but this file does **not** contain that fix.

Even stranger: my installation (`pipenv install cx_Freeze`) does contain a file
`"<HOME>\.virtualenvs\pyftpsync-wlHd8Q-f\Lib\site-packages\cx_Freeze\hooks.py"`
but again **without** the fix.
<img width="539" alt="image" src="https://user-images.githubusercontent.com/41663/177050127-3fb60227-5945-46cb-b753-11d0ccd16877.png">

(Maybe related: #418 )

**Desktop (please complete the following information):**
 - Platform information (e.g. Ubuntu Linux 22.04): Windows 10
 - OS architecture (e.g. amd64): amd64
 - cx_Freeze version [e.g. 6.11]: 6.11.1
 - Python version [e.g. 3.10]: 3.10

I don't really understand what's going on here :-/ Any suggestions?

**Comment by marcelotduarte** [comment]
Hi!
The hook for PyYaml implemented in #1124 (v6.7b7) was partially removed unintentionally in #1135 (v6.7b8).

> However this file does not exist in the current `cx_Freeze/` directory:
> https://github.com/marcelotduarte/cx_Freeze/tree/main/cx_Freeze

As you are using v6.11.1 hooks exists in that tree at: https://github.com/marcelotduarte/cx_Freeze/blob/v6.11.1/cx_Freeze/hooks.py

In 6.12-dev is in a new location.

To install the latest development build:
`pip install --pre --extra-index-url https://marcelotduarte.github.io/packages/ cx_Freeze`
If you are using conda-forge or msys2, download at: https://marcelotduarte.github.io/packages

**Comment by mar10** [comment]
FWIW:: I found that the build errors only occurred, when I explicitly list PyYAML as `includes` / `install_requires` / `setup_requires`.  
Since it is discoverable from the main package (now), I simply removed it, and everything works again.

Thanks for the quick response and the hard work on this tool!
